python-app-dependencies/pom.xml

```diff
@@ -16,27 +16,31 @@
 		<relativePath/>
 	</parent>
 
+	<properties>
+		<spring-cloud-starter-stream.version>2.1.0.BUILD-SNAPSHOT</spring-cloud-starter-stream.version>
```

Comment:
Should we use `project.version` instead? I'd like to understand how this could be different from the `project.version`